specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,130 @@
+- List item: `Invitations to events for contributors and surveys`
+- List item: `Announcements and changes about projects happening on Pontoon`
+- List item: `Changes and updates to the Pontoon platform`
+- Sub-header: `Click below to subscribe now`
+- Body: `Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform? This can be changed at any time in your settings.`
+- Subtitle: `You can update your email communication preferences at any time in your settings.`
+- Button: `No, thank you`
+- Button (primary): `Enable email updates`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Additional card for tour
+
+As part of new user onboarding, we point new users to user preferences and call out the ability to enable email communication preferences.
+
+(Section: content)
+- Header: `Stay up to date`
+- Body: `You can receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform by email. If you'd like to stay up to date, you can enable these emails in your settings.`
+
+*Display if user logged in:*
+- Button: `Enable email updates`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Unsubscribe page
+
+Legal regulations (e.g. CAN-SPAM act) have specific requirements to include an unsubscribe link in commercial email messages to manage email preferences without requiring a login.
+
+As a user I can:
+- Access my unique/personal unsubscribe page through a link without logging in. The link is typically included with non-transactional emails from Pontoon as an “Unsubscribe” link in the footer.
+  - Link technical specs:
+  1) Pontoon generates a random unique identifier and assigns it to each user. The algorithm doesn't use any existing user data to generate the ID, which needs to be sufficiently complex to avoid a potential attacker guessing.
+  2) The unsubscribe page can be accessed without login. If a user ID is not provided, or it's incorrect, it will display an error message.
+  3) Unsubscribe links in emails will include the user's unique identifier in the URL to correctly identify the user.
+- Access all options from a single page; I do not need to navigate from the page linked in the email to manage subscriptions.
+
+(Section: content)
+ - Title: `Mozilla Localization - Pontoon`
+ - Header: `Update Your Email Preferences`
+ - Body: `Thank you for contributing to localization at Mozilla. Please use the form below if you would like to stop receiving email updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform.`
+ - Radio button: `Please unsubscribe from all Mozilla localization emails.`
```

Comment:
```suggestion
 - Radio button: `Please unsubscribe me from all Mozilla localization emails.`
```

?

Reply:
We need to specify which radio option is automatically selected. On the other hand, what about a checkbox, automatically selected, with this text as a label?